engine/graphics/src/test/test_graphics.cpp

```diff
@@ -718,34 +716,32 @@ TEST_F(dmGraphicsTest, TestTextureAsyncDelete)
     params.m_Height = HEIGHT;
     params.m_Format = dmGraphics::TEXTURE_FORMAT_LUMINANCE;
 
+    const uint32_t TEXTURE_COUNT = 64;
```

Comment:
We now try to flood the thread(s) with more work to _really_ make sure this process works